src/pro_create.cpp

```diff
@@ -595,6 +595,8 @@ void update_buildrepair(void)
         STOP_WORKING(CH);
         int skill = GET_SKILL(CH, SKILL_CONJURING);
         int target = CH->char_specials.conjure[1];
+	int type = CH->char_specials.conjure[0];
+	aspect_conjuring_bonus(CH, CONJURING, type, target, skill);
```

Comment:
nit: indentation

If this is the same spot I put that nit last time, I bet you're using tabs while the file is using spaces, or vice/versa. The lack of consistency between those is frustrating AF with this codebase, and I've been switching to spaces wherever I can to avoid it!

Reply:
Think it's a software issue on my end, when I cut/copy/paste from the source into something else, then back, it appears to auto-indent instead of using spaces. I'll try to be mindful of it